### PR TITLE
Updated Demo: Setting up Drone CI with Bitbucket

### PR DESCRIPTION
**Pull request by majate**
We did some modifications to our video and have updated the video URL to match the latest version of our demo.

### CODE REVIEW

**Comment by andrebrogard** [comment]
Feedback:

## Strengths
* Great voice over
* Great video, it is very clear what is happening
* You explain actions very well

## To improve
* Pace too fast at times
* Clarify with voiceover what drone is (not just how it is setup).
* The provided ngrok URL seems sudden (because I'm not familiar with ngrok). Would love more clarification on it

**Comment by carinawic** [comment]
The video was well cut, with clear audio and proper subtitles. It had good explanations of what you were doing throughout the video. In the first "basic structure" slide, you have arrows explaining what the Drone entities are. However, you don't explain what BitBucket is. Instead of assuming that the viewer knows what BitBucket is, you could also give a short introduction to it. However, I understand that some assumptions must be made in regards to the time limit. Further, I feel that the video lacked a motivation to why this setup is necessary in regards to DevOps. Your take-home message is currently that we should be proud of our new CI setup - but why does a company need such a setup? I suggest that a short such motivation can be added in the very beginning of the video.

**Comment by Agriad** [comment]
# Feedback
### author: arieltan@kth.se

## Positive
* Great description  
The video contained a great description that will help users be able to follow along. It has a nice general description, documentation, and all steps and command of what is shown in the video.

* Nice animations  
The video contains some very nice animations that help make the video feel very smooth. It also helps keep the video flow. A great example with the animaiton is when it highlights certain parts of the screen with a zoom and slight contrast.

* Tidbits of information  
The popup in the corner is a great idea to show relevant information without using up time. It is also helpful to the relevant people but small enough for others to ignore it and focus on main message as well.

## Improvements
* Description typo  
There is a typo in the description with on the second paragraph. "Before following the seps shown in this demo" -> "Before following the steps shown in this demo" (typo in the word steps).

* Intro second page  
In the second page of the intro with Drone setup there is a small preview of the title. This can be a bit distraction and would probably be best if it is removed.

* Last page  
The last page can be very hard to read with the time limit and to time the pause correctly. Maybe it would be best to add into the description as well.

**Comment by elmaxe** [comment]
- Clear voice over
- Nice with background music
- Pretty and beautiful graphics
- Nice that the description contains detailed instructions with commands and code!

Improvements
- Perhaps you sometimes show more of the screen than necessary. Show what you currently are focusing on. For example at 0:50, you could have zoomed in on the "Permissions" part, so that it is easier to see what permissions you fill in
- The last thing you show before the video ends should perhaps be displayed for a bit longer. Right now it feels like when they in American medicine commercials display the negative effects of the medicine in one split second.


Good job! 😍

**Comment by m4reko** [comment]
Thank you for a really  nice demo! This is my feedback:

## Good points
- Clear voice over which is well timed with the content.
- Amazing with some [soul-less corporate music](https://www.youtube.com/watch?v=AIxY_Y9TGWI) (I mean it, it's awesome).
- Nice and clear graphics.

## Points of improvement
- At 0:51 it could be nice with a super short explanation on why those settings are set that way. I don't like following instructions blindly.
- I feel like it would be better to only show the bare minimum of windows at every moment in the clip for it to make sense. Now its a bit hard to follow what to focus on sometimes.
- I feel like this demo is leaning towards technicalities over context. I would prefer it to be the other way but I understand that it's a very hard balance when the time restriction is tight.

**Comment by jhammarstedt** [comment]
# Feedback for demo #1322 

## Positive

*  Great passing and tempo
*  Clear instructions and was easy to follow at most times
*  Nice music
*  Overall sublime video!

## Suggested improvements

* As the others noted the permissions at 0:50 could be showed a little more
* Sometimes having multiple windows up simultaneously made it harder to follow

Overall I think the video was great and you did a good job of demonstrating the subject in a clear way given the short time limit! 😊
